Background Scripts/Stale Tasks Auto-Close/Stale Tasks Auto-Close.js

```diff
@@ -0,0 +1,18 @@
+var staleDays = 7;
+var closeDays = 14;
+var reminderGR = new GlideRecord('task');
+reminderGR.addActiveQuery();
+reminderGR.addEncodedQuery('sys_updated_onRELATIVELE@dayofweek@ago@' + staleDays);
+reminderGR.query();
+while (reminderGR.next()) {
+    gs.eventQueue('task.reminder', reminderGR, reminderGR.assigned_to, staleDays + ' days without update.');
+    }
+
+var closeGR = new GlideRecord('task');
+closeGR.addActiveQuery();
+closeGR.addEncodedQuery('sys_updated_onRELATIVELE@dayofweek@ago@' + closeDays);
+closeGR.query();
+while (closeGR.next()) {
+    closeGR.state = 3; // Closed
```

Comment:
This may not be the state value for closed on some task extensions. Additionally, you should use setters (i.e. `setValue('field',value)`)

Reply:
Sure thanks for the suggestions @SapphicFire